.env.example

```diff
@@ -97,6 +97,7 @@ PERIODICAL=stop_expired_deploys:60,remove_expired_locks:60,report_system_stats:6
 # SLACK_CLIENT_ID= # optional, see plugins/slack_app/README.md
 # SLACK_CLIENT_SECRET= # optional, see plugins/slack_app/README.md
 # SLACK_VERIFICATION_TOKEN= # optional, see plugins/slack_app/README.md
+# SLACK_GLOBAL_BUDDY_REQUEST="<webhook_url>#<channel>" # send notifications for all buddy requests to this channel
```

Comment:
Maybe use the `?` character as that would be a natural URI break? I guess that would confuse things as well if you ever passed query params to the slack URL.